test_data/sdk_compiled_pipelines/valid/integration/ray_integration.py

```
@@ -1,16 +1,12 @@
 from kfp import compiler, dsl
 
-common_base_image = (
-    "registry.redhat.io/ubi8/python-39@sha256:3523b184212e1f2243e76d8094ab52b01ea3015471471290d011625e1763af61"
-)
-
 
 # image and the sdk has a fixed value because the version matters
-@dsl.component(packages_to_install=["codeflare-sdk==0.21.1"], base_image=common_base_image)
+@dsl.component(packages_to_install=["codeflare-sdk==0.32.2"], base_image='registry.access.redhat.com/ubi9/python-311:latest')
 def ray_fn() -> int:
     import ray  # noqa: PLC0415
```

Ruff (0.14.4)

Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)